spark-knn-core/src/main/scala/org/apache/spark/mllib/knn/KNNUtils.scala

```diff
@@ -10,5 +10,75 @@ object KNNUtils {
                            v2: Vector,
                            norm2: Double,
                            precision: Double = 1e-6): Double =
-    MLUtils.fastSquaredDistance(v1, norm1, v2, norm2, precision)
+    mllibMLUtilsfastSquaredDistance(v1, norm1, v2, norm2, precision)
+
+
+  // following two functions are copied to convert from mllib to ml
```

Comment:
Why copy these here? It seems `fastSquaredDistance` is still `private[mllib]` and should be accessible since we are at `org.apache.spark.mllib.knn`?